plugins/python-build/share/python-build/pypy2.7-7.3.8-src

```diff
@@ -0,0 +1,15 @@
+VERSION='7.3.8'
+PYVER='2.7'
+
+# https://www.pypy.org/checksums.html
+hash=0cdad270c62d3ccc53cc87eeb069a6dc46acaf95521b584624bcd6697d94fa1c
+
+### end of manual settings - following lines same for every download
+
+ver="pypy${PYVER}-v${VERSION}-src"
+url="https://downloads.python.org/pypy/${ver}.tar.bz2"
+
+#require_gcc
```

Comment:
No need to copypaste this long-commented-out code :wink:

Reply:
move shared bash code from python-build back into individual scripts
remove commented bash code